app/javascript/containers/screenings/PersonDemographicsFormContainer.js

```
@@ -33,7 +33,7 @@ const mergeProps = (stateProps, {dispatch}, {personId}) => {
       {
         dispatch(setField(personId, [field], value))
         dispatch(setField(personId, ['approximate_age'], null))
-        dispatch(setField(personId, ['approximate_age_units'], null))
+        dispatch(setField(personId, ['approximate_age_units'], 'years'))
```

Why the change in the default value for `approximate_age_units`? If the `date_of_birth` is set, then the `approximate_age` and `approximate_age_units` should both be `null`. 

`null` is a valid value for many of our data points. The code should work around `null` gracefully, not set arbitrary values in an effort to avoid `noFunction` errors.